application/lib/console.dart

```diff
@@ -1,29 +1,28 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:xterm/xterm.dart';
-import 'package:xterm/flutter.dart';
 
 import 'backend.dart';
 
 class Console extends StatefulWidget {
   const Console({
-    Key key,
-    @required this.game,
-    @required this.terminal,
+    Key? key,
+    required this.game,
+    required this.terminal,
   }) : super(key: key);
 
-  final CuddlyWorld game;
-  final Terminal terminal;
+  final CuddlyWorld? game;
+  final Terminal? terminal;
 
   @override
   _ConsoleState createState() => _ConsoleState();
 }
 
 class _ConsoleState extends State<Console> {
-  TextEditingController _input;
+  TextEditingController? _input;
```

Comment:
this is non-nullable. should either be initialized here or made late final

Reply:
fixed